lib/rubocop/cop/rspec/message_spies.rb

```diff
@@ -36,7 +36,7 @@ class MessageSpies < Cop
         SUPPORTED_STYLES = %w(have_received receive).freeze
 
         def_node_matcher :message_expectation, %(
-          (send (send nil :expect (send nil $_)) :to ...)
+          (send (send nil :expect $(...)) {:to :to_not :not_to} ...)
```

Comment:
I'll merge and fix it in a followup. No ned to go through a whole separate iteration of review.